qiskit/synthesis/discrete_basis/__init__.py

```diff
@@ -0,0 +1,91 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+r"""
+=======================================================================================
+Approximately decompose 1q gates to a discrete basis using the Solovay-Kitaev algorithm
+=======================================================================================
+
+.. currentmodule:: qiskit.synthesis.discrete_basis
+
+    Approximately decompose 1q gates to a discrete basis using the Solovay-Kitaev algorithm.
+
```

Comment:
The content under `.. currentmodule` shouldn't be indented - `currentmodule` is just a stateful directive - it doesn't take content.